bin/web/views/index.ejs

```diff
@@ -1,246 +1,341 @@
 <html>
+
```

Comment:
lots of whitespace changes unfortunately due to prettier, but i think the new spacing is more readable